From dbf356cd062d6313f90323a77d4cf7c820dae40f Mon Sep 17 00:00:00 2001
From: Ying Zhang <ying.zhang22455@nxp.com>
Date: Thu, 29 Sep 2016 13:51:32 +0800
Subject: [PATCH 123/124] armv8: aarch32: defconfig: Enable KVM-related
 configuration options

This patch is to enable KVM-related configuration options for host and
guest.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
---
 arch/arm/configs/ls_aarch32_defconfig |   10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/arch/arm/configs/ls_aarch32_defconfig
+++ b/arch/arm/configs/ls_aarch32_defconfig
@@ -102,7 +102,11 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=8
 CONFIG_BLK_DEV_RAM_SIZE=262144
+CONFIG_VIRTIO_BLK=y
 CONFIG_NETDEVICES=y
+CONFIG_TUN=y
+CONFIG_VIRTIO_NET=y
+CONFIG_VHOST_NET=y
 CONFIG_NET_VENDOR_FREESCALE is not set
 CONFIG_FSL_BMAN=y
 CONFIG_FSL_QMAN=y
@@ -130,6 +134,8 @@ CONFIG_SERIAL_8250=y
 CONFIG_SERIAL_8250_CONSOLE=y
 CONFIG_SERIAL_8250_EXTENDED=y
 CONFIG_SERIAL_8250_SHARE_IRQ=y
+CONFIG_SERIAL_AMBA_PL011=y
+CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
 CONFIG_SERIAL_OF_PLATFORM=y
 CONFIG_SERIAL_FSL_LPUART=y
 CONFIG_SERIAL_FSL_LPUART_CONSOLE=y
@@ -170,6 +176,7 @@ CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_DS3232=y
 CONFIG_DMADEVICES=y
 CONFIG_FSL_EDMA=y
+CONFIG_VIRTIO_PCI=y
 CONFIG_STAGING=y
 CONFIG_CLK_QORIQ=y
 # CONFIG_IOMMU_SUPPORT is not set
@@ -191,6 +198,7 @@ CONFIG_VFAT_FS=y
 CONFIG_NTFS_FS=m
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
+CONFIG_HUGETLBFS=y
 CONFIG_CONFIGFS_FS=y
 CONFIG_JFFS2_FS=y
 CONFIG_NFS_FS=y
@@ -221,3 +229,5 @@ CONFIG_CRC_CCITT=m
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=m
 CONFIG_LIBCRC32C=m
+CONFIG_VIRTUALIZATION=y
+CONFIG_KVM=y
